test/data/membership_with_keys.csv

```diff
@@ -0,0 +1,320 @@
+_key,_to,_from
```

Comment:
Can this file be made shorter? Or alternatively, is there a reason / lack of harm in leaving it this long?

Reply:
Good point, there's no real need for it to be this long. I'll truncate it.